internal/cache/controller_obj_cache.go

```diff
@@ -1931,12 +1931,16 @@ func (c *AviObjCache) PopulateL4PolicySetToCache(client *clients.AviClient, clou
 
 func (c *AviObjCache) AviPopulateAllStringGroups(client *clients.AviClient, cloud string, StringGroupData *[]AviStringGroupCache, nextPage ...NextPage) (*[]AviStringGroupCache, int, error) {
 	var uri string
-	akoUser := lib.AKOUser
 
 	if len(nextPage) == 1 {
 		uri = nextPage[0].NextURI
 	} else {
-		uri = "/api/stringgroup/?" + "&include_name=true&created_by=" + akoUser
+		//Fetching container specific StringGroups
```

Comment:
Using `uri = "/api/stringgroup/?" + "&include_name=true&name.contains=" + lib.GetAKOUser()` should be good enough? I think we are setting the stringgroup name in this format only.